Fast Pair Locator Tag sample: documentation update after integration of the FP Advertising Manager module (#23438)
nrfconnect/sdk-nrf: samples/bluetooth/fast_pair/locator_tag/README.rst

Fast Pair advertising policy
============================

The sample uses the :ref:`bt_fast_pair_adv_manager_readme` module and its trigger-based system to manage the advertising process.
The advertising is turned on if at least one trigger is active.

The sample code defines and manages the following triggers:

> * Pairing mode trigger, which occurs after a button action.

Reviewer: ```suggestion
* Pairing mode trigger that activates after a button action.
```

  By default, the trigger gets automatically enabled during the application bootup if the device is not provisioned.

> * DFU mode trigger, which occurs upon a DFU mode change.

Reviewer: ```suggestion
* DFU mode trigger that activates upon a DFU mode change.
```

The locator tag extension that is part of the :ref:`bt_fast_pair_adv_manager_readme` module additionally defines and manages the following triggers:

> * FMDN provisioning trigger, which occurs on the Account Key write operation during the FMDN provisioning operation.

Reviewer: ```suggestion
* FMDN provisioning trigger that activates on the Account Key write operation during the FMDN provisioning operation.
```

> * Beacon clock synchronization trigger, which occurs after the system bootup if the device is provisioned.

Reviewer: ```suggestion
* Beacon clock synchronization trigger that activates after the system bootup if the device is provisioned.
```

If the Fast Pair advertising is enabled, the sample selects the correct advertising mode based on the state of the pairing mode trigger:

   * Fast Pair discoverable advertising - Selected when the trigger is active.
   * Fast Pair not discoverable advertising - Selected when the trigger is inactive.

To fully disable Fast Pair advertising, all trigger requests must be removed.

> However, not all triggers can be manually disabled by the user, as the FMDN provisioning and Beacon clock synchronization triggers are managed by the :ref:`bt_fast_pair_adv_manager_readme` module automatically and are required for it to work correctly.

Reviewer: ```suggestion
However, you cannot manually disable all triggers, as the FMDN provisioning and Beacon clock synchronization triggers are managed by the :ref:`bt_fast_pair_adv_manager_readme` module automatically and are required for it to work correctly.
```
